<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>各种标准对话框实例</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="fileButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>文件标准对话框</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filelineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="colorButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>颜色标准对话框</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="colorLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="fontButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>字体标准对话框</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fontlineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="inputButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>标准输入对话框</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="udfButton">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>自定义消息对话框</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="udflineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="messageButton">
       <property name="text">
        <string>标准消息对话框</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
